obs_img_utils/cli.py

```diff
@@ -259,9 +261,28 @@ def packages():
     is_flag=True,
     help='Invoke packages REPL to specify package name filters'
 )
+@click.option(
+    '--output',
+    type=click.Choice(["text", "json"]),
```

Comment:
Nitpick: We've generally tried to default to single quotes in strings except where double quotes are required.

Reply:
fixed!